Packs/APIVoid/Integrations/APIVoid/README.md

```diff
@@ -0,0 +1,816 @@
+APIVoid wraps up a number of services such as ipvoid & urlvoid
+This integration was integrated and tested with version 01 of APIVoid
```

Comment:
is there an external link to the `01` we can reference?

Reply:
There is none, I wrote version 1, since at first I got an error for having XX.